src/main/java/io/reactivex/Flowable.java

```diff
@@ -7313,6 +7313,362 @@ public final Completable concatMapCompletableDelayError(Function<? super T, ? ex
         return RxJavaPlugins.onAssembly(new FlowableFlattenIterable<T, U>(this, mapper, prefetch));
     }
 
+    /**
+     * Maps the upstream items into {@link MaybeSource}s and subscribes to them one after the
+     * other succeeds or completes, emits their success value if available or terminates immediately if
+     * either this {@code Flowable} or the current inner {@code MaybeSource} fail.
+     * <p>
+     * <img width="640" height="305" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/concatMap.png" alt="">
+     * <dl>
+     *  <dt><b>Backpressure:</b></dt>
+     *  <dd>The operator expects the upstream to support backpressure and honors
+     *  the backpressure from downstream. If this {@code Flowable} violates the rule, the operator will
+     *  signal a {@code MissingBackpressureException}.</dd>
```

Comment:
[signal]**s**

Reply:
Same in all javadocs below

Reply:
Sounds fine with me:

"the operator **will** signal a MissingBackpressureException."

Reply:
wow, sure, my bad, sorry